### PR TITLE
Feature/eco counter

### PR DESCRIPTION
**Pull request by jankosk**
Initial version for eco counters. There are still some missing Swedish translations and workarounds for missing data e.g. counts data for 'Oulu_kapy' and 'Oulu_ELY' sites.

### CODE REVIEW

File: app/component/map/tile-layer/TileLayerContainer.js [hunk 1]
```diff
@@ -45,13 +46,23 @@ import TileContainer from './TileContainer';
...
   selectableTargets: undefined,
   coords: undefined,
   showSpinner: true,
 };
 
+// TODO: Add other domains when site data is available.
+const getSiteIdFromChannelId = (id, domain) => {
+  if (domain === 'Oulu_kaupunki') {
+    return `${id.slice(0, 2)}0${id.slice(3, id.length)}`;
+  }
```
Comment (jankosk) [inline app/component/map/tile-layer/TileLayerContainer.js]: I'm assuming here that the third character of the`siteId` in 'Oulu_kaupunki' domains always denotes the different direction and user types, and that 0 is always the "base" site. I'm not sure if this is always the case. Also do the other domain siteIds work similarly?

File: app/component/map/popups/EcoCounterPopup.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            icon="icon-icon_eco-counter"
+            unlinked
+          />
+          <Relay.Renderer
+            Container={EcoCounterContent}
+            queryConfig={
+              new EcoCounterDataRoute({
+                outId: channels[0].siteId,
+                inId: channels[1].siteId,
+                domain: ecoCounterSite.domain,
```
Comment (jankosk) [inline app/component/map/popups/EcoCounterPopup.js]: I'm assuming here that the first one in channels array for is "out" direction and the second one is "in" direction. Not sure if this is the case.

File: app/component/map/tile-layer/TileLayerContainer.js (hunk 1 above)
Comment (kryysler) [reply app/component/map/tile-layer/TileLayerContainer.js]: There is no guarantee that this is the case. The "user type", i.e. pedestrian or bicycle is contained in it's own property `userType` for each channel. The direction however is not specified directly anywhere.

Comment (kryysler) [reply app/component/map/tile-layer/TileLayerContainer.js]: After reviewing the documentation at http://eco-test2.com/apidoc/wso2/apidoc.html, it says the `sens` property includes the direction; "Direction detection information. {1: IN direction, 2: OUT direction, 5: No direction distinction}".

File: app/component/map/popups/EcoCounterPopup.js (hunk 2 above)
Comment (kryysler) [reply app/component/map/popups/EcoCounterPopup.js]: See comment above.

**Comment by jankosk** [comment]
Added missing translations and removed unnecessary ecoCounterSites query. It should now work even if the direction is not specified and if only walking or cycling data is available.

**Review by kryysler — APPROVED** [review]

**Pull request merged**